src/common.jl

```diff
@@ -0,0 +1,70 @@
+abstract type TaylorAlgorithm <: DEAlgorithm end
+struct TaylorMethod <: TaylorAlgorithm
+    order::Int
+end
+
+TaylorMethod() = error("Maximum order must be specified for the Taylor method")
+
+export TaylorMethod
+
+function DiffEqBase.solve{uType,tType,isinplace,AlgType<:TaylorAlgorithm}(
+    prob::AbstractODEProblem{uType,tType,isinplace},
+    alg::AlgType,
+    timeseries=[],ts=[],ks=[];
+    verbose=true, abstol = 1e-6, save_start = true,
```

Comment:
I suggest to have `abstol = 1e-20` as default value, or smaller, to actually get the accuracy that the method allows. 

`abstol` is used to define the step-size in each iteration, and corresponds to the size of the last (or one-before-last) term of the local Taylor expansion. Having it very small is what makes the error (per integration step) to be the round-off error.

Reply:
In here:

http://docs.juliadiffeq.org/latest/basics/common_solver_opts.html#Basic-Stepsize-Control-1

it states that the universal default (for ODEs at least) is `abstol=1e-6`. This is what every other package (even the old C++ and Fortran ones) tend to do anyways, so it has precedence. In the docs part mentioning this package I mentioned it should be used for high accuracy calculations, but I think that it should keep the common default just for consistency.

Reply:
I get your point, and agree that a comment in the docs is always worth, and what you wrote is quite clear on that.

Yet, I prefer to have "useful" defaults (as user) rather than standardized ones. Having `abstol = 1e-6` makes me specifying it every time (because I use `TaylorIntegration` for accurate results); having it as `1e-20` allows me to skip it, unless I want even more accuracy.

Reply:
It is a useful default though. Most people solving an ODE don't want or need `abstol = 1e-6`. Only in the case of accurate solving does something like `absol = 1e-20` make sense. `absol = 1e-20` could never be the global default since it would also just break in a lot of cases due to floating point error.

Reply:
Just a quick comment: I agree with you that for most people `abstol = 1e-6` is enough, but they will not be using `TaylorIntegration.jl`. If they use `TaylorIntegration.jl` is because they do want to push the accuracy to its limits, and in that case having `abstol = 1e-6` will generate a step-size that is probably so large that the results are not so close to the exact solution. That's the reason I think that `abstol = 1e-20` is reasonable for TaylorIntegration, whereas for other used methods a reasonable default is `1e-6`, as it is already.